pyproject.toml

```diff
@@ -1,8 +1,7 @@
-[tool.poetry]
+[project]
 name = 'expandvars'
-version = '0.11.0'
+version = '0.12.0'
```

Comment:
I think that instead of keeping version in two places, you could do:

```toml
[project]
dynamic = ["version"]

[tool.hatch.version]
path = "expandvars.py"
```

See https://hatch.pypa.io/latest/config/metadata/#version; https://hatch.pypa.io/latest/version/